src/topology/metric_space/gromov_hausdorff_realized.lean

```diff
@@ -243,8 +243,11 @@ begin
                ∩ (⋂x y, {f : Cb X Y | f (x, y) = f (y, x)})
                ∩ (⋂x y z, {f : Cb X Y | f (x, z) ≤ f (x, y) + f (y, z)})
                ∩ (⋂x, {f : Cb X Y | f (x, x) = 0})
-               ∩ (⋂x y, {f : Cb X Y | f (x, y) ≤ max_var X Y}) :=
-    begin ext, unfold candidates_b, unfold candidates, simp [-sum.forall], refl end,
+               ∩ (⋂x y, {f : Cb X Y | f (x, y) ≤ max_var X Y}),
+  { ext,
+    simp only [candidates_b, candidates, mem_inter_eq, mem_Inter, mem_set_of_eq,
+               continuous_map.to_fun_eq_coe],
```

Comment:
Does changing
```
private def candidates_b : set (Cb X Y) := {f : Cb X Y | f.to_fun ∈ candidates X Y}
```
to
```
private def candidates_b : set (Cb X Y) := {f : Cb X Y | f ∈ candidates X Y}
```
remove the need for this last lemma in the simp set?

Reply:
Yes, better. Done.